jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1657,16 +1654,24 @@ def _get_min_identity(tf_dtype):
 tf_impl_with_avals[lax.reduce_window_p] = _reduce_window
 # pylint: enable=protected-access
 
-# We use lax_control_flow._cumred_tpu_translation_rule to convert cummin and
-# cummax. This is efficient on TPU, but the complexity is O(n^2) on other
-# backends. This may be implemented using associative_scan instead to favor
-# different backends.
+# We use lax_control_flow._cumred_tpu_translation_rule to convert cummax,
+# cummin, cumsum and cumprod. This is efficient on TPU, but the complexity is
+# O(n^2) on other backends. This may be implemented using associative_scan
+# instead to favor different backends.
 tf_impl_with_avals[lax_control_flow.cummin_p] = _convert_jax_impl(
     functools.partial(lax_control_flow._cumred_tpu_translation_rule,
                       lax._reduce_window_min), multiple_results=False)
 tf_impl_with_avals[lax_control_flow.cummax_p] = _convert_jax_impl(
     functools.partial(lax_control_flow._cumred_tpu_translation_rule,
                       lax._reduce_window_max), multiple_results=False)
+# TODO(bchetioui): cumsum and cumprod can be converted using pure TF ops for
```

Comment:
Since this is fresh in your mind now, can you add a bit more detail here, e.g., which dtypes can be implemented?